Gifski/CheckerboardView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  CheckerboardView.swift
+//  Gifski
+//
+//  Created by Sergey Kuryanov on 06/10/2019.
+//  Copyright © 2019 Sindre Sorhus. All rights reserved.
+//
+
+import Cocoa
+
+class CheckerboardView: NSView {
+	let gridSize = CGSize(width: 10, height: 10)
+	let firstColor = NSColor.white
+	let secondColor = NSColor.lightGray
```

Comment:
This is a bit too dark. I would use `#d1d1d1` (preferably a semantic color close to this).